src/Network/Network/Generated/Models/PSExpressRouteCircuit.cs

```diff
@@ -46,7 +46,7 @@ public partial class PSExpressRouteCircuit : PSTopLevelResource
         public string ProvisioningState { get; set; }
         public string GatewayManagerEtag { get; set; }
         [Ps1Xml(Target = ViewControl.Table)]
-        public bool? AllowGlobalReach { get; set; }
```

Comment:
Removing this property is a breaking change: https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#cmdletoutputbreakingchangeattribute

Reply:
It's a dead property. We added this property to swagger initially, and then abandoned afterwards. I've updated breakingchanges.csv

Reply:
@syfarogh Sorry, but you cannot remove properties. Removing properties is a breaking change. This was added 8 months ago and has been out in the wild. So, there may be scripts that check this value. We cannot allow those scripts to be broken, even if the property has no purpose. Please keep the property and use our breaking change attribute: https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#a-few-properties-in-the-output-type-are-being-deprecated

Reply:
Below is the PR where I added this field in swagger.
https://github.com/Azure/azure-rest-api-specs/pull/3542

Following PR I had created to consume the field but then we abandoned in favor of the new field (GlobalReachEnabled) being added in this PR.
https://github.com/Azure/azure-powershell/pull/6845

We don't have any scripts on the flag, so removal will be safe. This will align PS model with swagger model. can it be removed?